modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -325,7 +329,7 @@ class DisplayElements extends Component {
         );
       }
       return row;
-    });
+    }.bind(this));
```

Comment:
`this` shouldn't have to be bound to the render function.

Reply:
See the comment from @zaliqarosli and I removed it from the render function.